homeassistant/components/august/camera.py

```diff
@@ -64,12 +64,14 @@ def model(self):
 
         if self._image_url is not latest.image_url:
             self._image_url = latest.image_url
-            self._image_content = requests.get(
-                self._image_url, timeout=self._timeout
-            ).content
+            self._image_content = await self.hass.async_add_job(self._camera_image)
 
         return self._image_content
 
+    def _camera_image(self):
+        """Return bytes of camera image via http get."""
+        return requests.get(self._image_url, timeout=self._timeout).content
```

Comment:
Normally we don't allow this code in home assistant. It should really be moved to the library.

Reply:
I'll work on a get_camera_image function for this in py-august.  I'll resubmit this for now with change to async_add_executor_job and add a note that its being replaced

Created an issue to track https://github.com/home-assistant/home-assistant/issues/32048

Reply:
https://github.com/snjoetw/py-august/pull/41/commits/1e86023f9254699bd3bb994cace79b775f2d9d1f

Upstream PR for reference

Reply:
https://github.com/snjoetw/py-august/pull/41